website/docs/docs/deploy/job-commands.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Job commands"
+id: "job-commands"
+description: "How to use dbt commands to set tasks for your dbt Cloud jobs."
+---
+
+A dbt Cloud production job allows you to set up a system to run a dbt job and job commands on a schedule, rather than running dbt commands manually from the command line or [IDE](/docs/get-started/develop-in-the-cloud). A job consists of multiple commands, and their combined outcomes determine the job's run status, which can be successful, canceled, or failed. During a job run, each command outcome can either succeed or fail under the **Run steps** section.
+
+Each job allows you to:
+
+- Configure job commands
+- View job run details, including timing, artifacts, and detailed run steps
+- Access logs to view or help debug issues and historical invocations of dbt
+- Set up notifications, and [more](/docs/deploy/deployments#dbt-cloud)!
+
+## Job command types
+
+Job commands are specific tasks executed by the job, and you can configure them seamlessly by either adding [dbt commands](/reference/dbt-commands) or using the checkbox option in the **Commands** section. 
+
+During a job run, the commands are "chained" together, and you can expect different outcomes when adding the command as a run step compared to selecting a checkbox.
```

Comment:
Unless what's being said here is obvious to our customers, this sentence feels a little vague. As an outsider reading about this feature, I'm not sure what it's trying to express.

Reply:
gotcha! we're trying to let the user know that commands are linked and there are diff outcomes depending on the type of command you use (checkbox, manually adding commands). rephrased here:

During a job run, the commands are "chained" together and executed as run steps. When you add a dbt command in the **Commands** section, you can expect different outcomes compared to using the checkbox option.